roles/splunk_cluster_master/tasks/configure_indexes.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Add stanza for the index specified
+  ini_file:
+    dest: "{{ splunk.home }}/etc/master-apps/_cluster/local/indexes.conf"
+    section: "{{ index.index }}"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+  with_items:
+    - { key: "remotePath", value: "volume:{{ index.remoteName }}/$_index_name" }
+    - { key: "repFactor", value: "auto" }
+
+- name: Add path for the remote specified
+  ini_file:
+    dest: "{{ splunk.home }}/etc/master-apps/_cluster/local/indexes.conf"
+    section: "volume:{{ index.remoteName }}"
+    option: "{{ item.key }}"
+    value:  "{{ item.value }}"
+  with_items:
+    - { key: "storageType", value: "remote" }
+    - { key: "path", value: "{{ index.scheme }}://{{ index.remoteLocation }}" }
+
+- name: Write AWS credentials for the remote specified
+  ini_file:
+    dest: "{{ splunk.home }}/etc/master-apps/_cluster/local/indexes.conf"
+    section: "volume:{{ index.remoteName }}"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+  with_items:
```

Comment:
I don't think we've tested this yet, but I think it would be nice to support native IAM access (without plaintext credentials being exposed). 

For this case, I would assume:
1) The cluster is already running in AWS
2) The worker nodes have the correct IAM privileges to access the S3 bucket/volume
3) The containers/pods have access to the IAM privileges (via host networking or other proxying means)